week-3/Homework/mandatory/1-oxygen-levels.js

```diff
@@ -7,34 +7,56 @@ Their computer detects a list of nearby planets that have Oxygen in their atmosp
 To be safe, they need to land on the first unamed planet that has Oxygen levels between 19.5% and 23.5%.
 
 Write a function that finds the oxygen level of the first safe planet - Oxygen between 19.5% and 23.5%
+
+Some string methods that might help you here are .replace() and .substring(). Let's look at a quick
+example before trying the exercise.
 */
 
-function safeLevels() {
+/* .replace() allows us to add something where we removed something*/
+// let oxygenLevels1 = ["24.2%", "11.3%", "19.9%", "23.1%", "29.3%", "20.2%"];
+// let oxygenLevels2 = ["20.2%", "23.1%"];
+// function oxygenLevels1(level);
+
+// return level.replace(3,5);
+
+/* .substring() allows us to remove things from strings */
+// let oxygenLevels1
+
+// console.log(oxygenLevels1)
 
+/* + + + + + + + */
+/* Now try the exercise */
+
+function safeLevels(Levels) {
+	let safeLevels1 = Levels.find(function (goodexygen) {
+		return goodexygen > "19.5%" && goodexygen < "23.5%";
```

Comment:
This function is good!

You just need to do the correct spelling with 

`goodexygen` to `goodOxygen`